config/env_sample.json

```diff
@@ -206,6 +206,25 @@
     "/*  DATA_WAREHOUSE_PORT=5432": "*/",
     "/*  Enable/Disable Unizin Date Warehouse specific features/data": "*/",
     "DATA_WAREHOUSE_IS_UNIZIN": true,
+    "/*  Learning Record Store configuration": "*/",
+    "/*  Uncomment these variables and fill them in if you're using a ": "*/",
+    "/*  non-bigquery LRS source. These are optional": "*/",
+    "/*  LRS database engine driver": "*/",
+    "LRS_ENGINE": "django.db.backends.postgresql",
```

Comment:
Looking at this, this would also probably be better if we had this as an object similar to the settings like

```
"LRS": {
  "ENGINE": . . .
```

That probably should be a separate issue though.

Reply:
I'm up for doing the change but if doing it for `LRS_*` we should also do it for `DATA_WAREHOUSE_*` and `MYSQL_*`

I can make a separate issue+PR after this is merged

Reply:
See #1220